scripts/release.sh

```diff
@@ -83,7 +83,7 @@ fi
 mvn org.codehaus.mojo:versions-maven-plugin:2.7:set -DnewVersion=${VERSION} -DgenerateBackupPoms=false
 
 if [[ "${SUFFIX}" = "dependencies" ]]; then
-  sed -i "" "s/version = .*/version = ${VERSION}/" gradle-plugin/gradle.properties
```

Comment:
It failed with `sed: can't read s/version = .*/version = 1.5.4/: No such file or directory`